packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -21,3 +21,14 @@ export const sync = () => ({
       'Apply synchronization when there is a 100% match between organization and snapshot resources.',
   }),
 });
+
+export const previewLevel = () => ({
+  previewLevel: flags.enum({
+    char: 'p',
+    description:
+      'The verbosity of the preview. The `light` preview is faster to generate but only contains a limited amount of information, as opposed to the `detailed` preview which takes more time to generate, but returns a diff representation of all the changes to apply.',
```

Comment:
@jpmarceau can you let me know if this description makes sense?

Reply:
Yep I think that's quite good 👍